.github/workflows/lintercheck.yml

```diff
@@ -0,0 +1,79 @@
+name: Linter check
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags:
+      - r[0-9]+.[0-9]+
+
+jobs:
+  linter_check:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repo
```

Comment:
nit: Using our generic jobs can reduce the need for a lot of this boilerplate - https://github.com/pytorch/test-infra/wiki/Writing-generic-CI-jobs. See vision's lint job for example: https://github.com/pytorch/vision/blob/main/.github/workflows/lint.yml

Reply:
I'll try using it for builds, but linter I want to run on stock runners